### ISSUE TITLE
[Feature request] Version Bump :)

### ISSUE THREAD

**Issue by cocoonkid** [haidaraM/ansible-playbook-grapher#154]
```
Using version ^2.0.0 for ansible-playbook-grapher

Updating dependencies
Resolving dependencies... (1.6s)

Because no versions of ansible match >8.0.0,<8.1.0 || >8.1.0,<8.2.0 || >8.2.0,<8.3.0 || >8.3.0,<8.4.0 || >8.4.0,<9.0.0
 and ansible (8.0.0) depends on ansible-core (>=2.15.0,<2.16.0), ansible (>=8.0.0,<8.1.0 || >8.1.0,<8.2.0 || >8.2.0,<8.3.0 || >8.3.0,<8.4.0 || >8.4.0,<9.0.0) requires ansible-core (>=2.15.0,<2.16.0).
And because ansible (8.1.0) depends on ansible-core (>=2.15.1,<2.16.0)
 and ansible (8.2.0) depends on ansible-core (>=2.15.2,<2.16.0), ansible (>=8.0.0,<8.3.0 || >8.3.0,<8.4.0 || >8.4.0,<9.0.0) requires ansible-core (>=2.15.0,<2.16.0).
And because ansible (8.3.0) depends on ansible-core (>=2.15.3,<2.16.0)
 and ansible (8.4.0) depends on ansible-core (>=2.15.4,<2.16.0), ansible (>=8.0.0,<9.0.0) requires ansible-core (>=2.15.0,<2.16.0).
Because no versions of ansible-playbook-grapher match >2.0.0,<3.0.0
 and ansible-playbook-grapher (2.0.0) depends on ansible-core (>=2.12,<2.15), ansible-playbook-grapher (>=2.0.0,<3.0.0) requires ansible-core (>=2.12,<2.15).
Thus, ansible-playbook-grapher (>=2.0.0,<3.0.0) is incompatible with ansible (>=8.0.0,<9.0.0).
So, because severaln1nes depends on both ansible (^8.0.0) and ansible-playbook-grapher (^2.0.0), version solving failed.
```

**Comment by haidaraM** [comment]
Hmm interesting. At first glance, this [PR](https://github.com/haidaraM/ansible-playbook-grapher/pull/151/files) should fix this. Will see what I can do in the following days. 

In the meantime, I suggest you install the grapher in a virtualenv with no other conflicting dependencies.

**Comment by haidaraM** [comment]
Fixed in 2.1.0
